addons/web_editor/static/tests/html_field_tests.js

```diff
@@ -379,6 +379,8 @@ QUnit.module("WebEditor.HtmlField", ({ beforeEach }) => {
                 route === `/web_editor/modify_image/${imageRecord.id}`
             ) {
                 if (modifyImageCount === 0) {
+                    assert.ok(args.res_model);
+                    assert.ok(args.res_id);
```

Comment:
Wouldn't it be better to test the actual values rather ?

Reply:
Indeed. Done!